ci/k6-ingestion-test.js

```diff
@@ -1,6 +1,17 @@
 import http from 'k6/http'
-import {check, group, sleep} from 'k6'
+import {check, sleep, fail} from 'k6'
 import { Gauge } from 'k6/metrics'
+import { URL } from 'https://jslib.k6.io/url/1.0.0/index.js';
+
+const POSTHOG_API_ENDPOINT = __ENV.POSTHOG_API_ENDPOINT
+const POSTHOG_EVENT_ENDPOINT = __ENV.POSTHOG_EVENT_ENDPOINT
+
+const captureURL = new URL(`${POSTHOG_EVENT_ENDPOINT}/e/`);
```

Comment:
Note: There's now very inconsistent semi-colon usage through the file.

Worth adding eslint to the project to autoformat this or just removing?